schema/swagger.yml

```diff
@@ -179,32 +179,36 @@ definitions:
   NodeStatus:
     type: object
     required:
-      - syncing
+      - broadhash
+      - consensus
       - height
+      - loaded
       - networkHeight
-      - consensus
-      - broadhash
+      - syncing
     properties:
-      syncing:
-        type: boolean
-        example: false
+      broadhash:
+        type: string
+        example: 258974416d58533227c6a3da1b6333f0541b06c65b41e45cf31926847a3db1ea
+        minLength: 64
+        maxLength: 64
+      consensus:
+        type: integer
+        example: 95
+        minimum: 0
+        maximum: 100
       height:
         type: integer
         example: 123
         minimum: 1
+      loaded:
+        type: boolean
+        example: true
```

Comment:
Did you confirmed from @Isabello for this new attribute? Since she is need to give go ahead on final draft for swagger api.

Reply:
Thanks, I haven't noticed her about it. Please @Isabello this should be updated in the next API version.

Reply:
Loaded can be in there, just ensure the swagger spec includes it on the hub